### ISSUE TITLE
 Can anyone help me?

### ISSUE THREAD

**Issue by iszotak** [virtacoin/VirtaCoinProject#9]
Hi, 
I have problem transfer to my wallet, can anyone help me my wallet is locked un able to
importprivkey.

Thanks,
Irene

iszotak@gmail.com

**Comment by MONEYMAN1298** [comment]
It seems like an easy fix. I'm not that technical, so Join the online Reddit VirtaCoin Community at http://www.reddit.com/r/VirtaCoin and post your question there. They will be able to answer it.

**Comment by pranto** [comment]
Yes i can help  you if you have skype join to my skype i will help you . you just give me your skype id i will add and tel you how to do

Date: Thu, 3 Jul 2014 08:56:47 -0700

**Comment by prabumr** [comment]
https://www.atomic-trade.com/markets
Here the value of VTA is low But the value of VTA is high how we work together to do
yesterday the site shows VTA/BTC = 0.00001
Now it shows VTA/BTC = 0.0000001
we need to increase the value of VTA so long time waiting it worthless means not good so we work together to increase of 1 VTA = 1 BTC
http://virtacoin.do.am/forum/25-7-1
